### PR TITLE
Skale tipping miniapp docs

### PR DESCRIPTION
**Pull request by iSpeakNerd**
resolves https://github.com/abridged/collabland-monorepo/issues/2576

### CODE REVIEW

**Review by gitaalekhyapaul — APPROVED** [review]
LGTM :+1:

File: help-docs/marketplace/apps/give.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import img1 from '@site/static/img/tutorial/marketplace/poap.png';
+
+## Overview
+
+Send and receive tokens on the SKALE blockchain.
+
+### About this app
+
+Give allows community members to give and receive tokens onchain on the SKALE blockchain. Tokens are sent between members' [gmPASS-controlled smart accounts](/help-docs/key-features/gm-pass), creating a personal web of onchain connections between community members.
```
Comment (gitaalekhyapaul) [inline help-docs/marketplace/apps/give.md:16]: Tokens are sent between members' custodial wallets

**Review by gitaalekhyapaul — CHANGES_REQUESTED** [review]

File: help-docs/marketplace/apps/give.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+<!-- 1. Install the Give Miniapp from the [Collab.Land Marketplace](https://cc.collab.land).
+
+2. -->
+
+### Commands available:
+
+There is one command `/give` with four total subcommands.
+
+| Command | Additional Parameters | Description |
+| --- | --- | --- |
```
Comment (gitaalekhyapaul) [inline help-docs/marketplace/apps/give.md:31]: There's some issue in the table rendering:
image

**Review by gitaalekhyapaul — CHANGES_REQUESTED** [review]

**Comment by iSpeakNerd** [comment]
- [ ] add [skale link](https://skale.space/)

File: help-docs/marketplace/apps/give.md (hunk 2 above)
Comment (iSpeakNerd) [reply help-docs/marketplace/apps/give.md:31]: what is the issue?

Comment (iSpeakNerd) [reply help-docs/marketplace/apps/give.md:31]: what is the issue @gitaalekhyapaul?

File: help-docs/marketplace/apps/give.md (hunk 1 above)
Comment (iSpeakNerd) [reply help-docs/marketplace/apps/give.md:16]: @raymondfeng please clarify here

File: help-docs/marketplace/apps/give.md (hunk 2 above)
Comment (iSpeakNerd) [reply help-docs/marketplace/apps/give.md:31]: ohhhhhhhhhhhhhh its actually displaying `---------------` is the issue nm

Comment (gitaalekhyapaul) [reply help-docs/marketplace/apps/give.md:31]: There's the top row in the table with a bunch of `-`

File: help-docs/marketplace/apps/give.md (hunk 1 above)
Comment (iSpeakNerd) [reply help-docs/marketplace/apps/give.md]: I commented out the whole section. can add back later if desired

**Review by gitaalekhyapaul — APPROVED** [review]
LGTM :+1:

**Pull request merged**